config/initializers/content_security_policy.rb

```diff
@@ -1,25 +1,17 @@
-# Be sure to restart your server when you modify this file.
+Rails.application.config.content_security_policy do |policy|
+  policy.default_src :self
+  policy.font_src    :self, :https, :data
+  policy.img_src     :self, :https, :data
+  policy.object_src  :none
+  policy.font_src    :self, 'fonts.gstatic.com'
+  policy.connect_src :self
+  policy.script_src  :self, :unsafe_eval
```

Comment:
SUGGESTION: You can likely just do :unsafe_eval if the env is `development`, pretty sure in a production setting it SHOULD work right :)

Reply:
I think we actually have to do this because jquery uses eval in a bunch of places! Unfortunately. I'm curious as to whether there's a way to get jquery out but that is another issue for another time probably.